notebooks/demo.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    text_splitter = CharacterTextSplitter(chunk_size=1000, chunk_overlap=0)
</pre><p> </p><p>I believe that a better split could be achieved by using the MarkdownTextSplitter instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Thanks for pointing this out! 
The `chunk_size` parameter would be a hardcoded value in the MarkdownTextSplitter as well. Do we yet know of a way to better decide that value?